jc3mp.kvp

```diff
@@ -0,0 +1,87 @@
+Meta.DisplayName=Just Cause 3 Multiplayer Mod
+Meta.Description=Just Cause 3 Multiplayer Dedocated Server
+Meta.OS=Windows, Linux
+Meta.Arch=x86_64
+Meta.Author=ThisIslandEarth
+Meta.URL=https://just-cause.mp
+Meta.DisplayImageSource=steam:619910
+Meta.EndpointURIFormat=
```

Comment:
To allow the Connect button to work this should be:
`Meta.EndpointURIFormat=steam://connect/{ip}:{GenericModule.App.Ports.$queryPort}`

Reply:
Added